app/src/main/java/org/thoughtcrime/securesms/groups/EditClosedGroupActivity.kt

```diff
@@ -283,18 +301,19 @@ class EditClosedGroupActivity : PassphraseRequiredActionBarActivity() {
         }
 
         if (members.isEmpty()) {
-            return Toast.makeText(this, R.string.activity_edit_closed_group_not_enough_group_members_error, Toast.LENGTH_LONG).show()
+            return Toast.makeText(this, R.string.groupCreateErrorNoMembers, Toast.LENGTH_LONG).show()
         }
 
         val maxGroupMembers = if (isClosedGroup) groupSizeLimit else legacyGroupSizeLimit
         if (members.size >= maxGroupMembers) {
-            return Toast.makeText(this, R.string.activity_create_closed_group_too_many_group_members_error, Toast.LENGTH_LONG).show()
+            return Toast.makeText(this, R.string.groupAddMemberMaximum, Toast.LENGTH_LONG).show()
         }
 
         val userPublicKey = TextSecurePreferences.getLocalNumber(this)!!
         val userAsRecipient = Recipient.from(this, Address.fromSerialized(userPublicKey), false)
 
         if (!members.contains(userAsRecipient) && !members.map { it.address.toString() }.containsAll(originalMembers.minus(userPublicKey))) {
+            // ACL TODO - Need a proper string for this
             val message = "Can't leave while adding or removing other members."
```

Comment:
Adding a note here to make sure this gets addressed

Reply:
Changed toast to warning - there's no way to get into this state through the UI (although potentially there was - hence someone wrote this check).